### ISSUE TITLE
using without CanvasInput.js

### ISSUE THREAD

**Issue by arpu** [shaneharris/aframe-material-collection#3]
is it possible to use CanvasInput.js optional?

**Comment by sidequestlegend** [comment]
Yes canvasinput can be excluded if you are not using the text input fields and you import the components separately. I will make it optional in the compiled version too. I plan to improve the canvasinput and remove the dom dependencies and fix a few other bugs. Are you looking for an alternative text input option?

**Comment by arpu** [comment]
maybe this is an alternative methode https://github.com/etiennepinchon/aframe-material/blob/master/src/input/index.js

**Comment by sidequestlegend** [comment]
Yeh I had looked at that implementation and i love his keyboard. That one didnt seem to support a normal keyboard and i wanted this UI kit to be as open to 2d mode and regular input as much as possible. 

I may take the keyboard he has though, it looks pretty solid. 

In the mean time I have re-implemented the text input and I have built my own text input based on a-text which seems to work really well so far and supports things like shortcuts ( ctrl+a, shift + arrows ) and  mouse interaction. I have tried to match browser behavior as much as i can. I do plan to add more features but for now its stable enough to use. 

The old CanvasInput.js based input is still there but deprecated and all the examples now use the new input. Ill close this since canvasinput is no longer needed for inputs.
